requirements.txt

```diff
@@ -1,8 +1,11 @@
-build==1.2.1
-certifi==2024.2.2
-charset-normalizer==3.3.2
-idna==3.7
-packaging==24.0
-pyproject_hooks==1.0.0
-requests==2.31.0
-urllib3==2.2.1
+build==1.2.1; python_version >= '3.8'
+certifi==2024.2.2; python_version >= '3.8'
+charset-normalizer==3.3.2; python_version >= '3.8'
+idna==3.7; python_version >= '3.8'
+packaging==24.0; python_version >= '3.8'
+pyproject_hooks==1.1.0; python_version >= '3.8'
+requests==2.31.0; python_version >= '3.8'
+urllib3==2.2.1; python_version >= '3.8'
+importlib_metadata==7.1.0; python_version == '3.8' or python_version == '3.9'
+tomli==2.0.1; python_version == '3.8' or python_version == '3.9' or python_version == '3.10'
+zipp==3.18.1; python_version == '3.8' or python_version == '3.9'
```

Comment:
Are those not needed for Python > 3.9 (> 3.10 in the case of `tomli`)?

Reply:
yes, python 3.10 and higher requires only first 8